src/models/pieChart.js

```diff
@@ -208,8 +211,8 @@ nv.models.pieChart = function() {
         dispatch.tooltipShow(e);
     });
 
-    dispatch.on('tooltipShow', function(e) {
-        if (tooltips) showTooltip(e);
+    pie.dispatch.on('elementMouseout.tooltip', function(e) {
+        dispatch.tooltipHide(e);
     });
 
     dispatch.on('tooltipHide', function() {
```

Comment:
I am actually copying the style from the lineChart for better consistency with the other charts:
https://github.com/nvd3-community/nvd3/blob/development/src/models/lineChart.js#L360

I can change the position of this block of code in all the graphs as you are saying, but it'll have to wait until next week when I come back from vacation and get access to a proper notebook.